src/utils/style-formatter.js

```diff
@@ -45,7 +46,10 @@ export default {
       bgUrl = bgUrl.replace(/'/g, "\\'");
       styles += formatProperty('background-image', `url('${bgUrl}')`);
       styles += formatProperty('background-size', backgroundSize[style.background.size] || backgroundSize.auto);
-      styles += formatProperty('background-position', backgroundPosition[style.background.position] || backgroundPosition.topLeft);
+      styles += formatProperty(
```

Comment:
This was auto formatted. But do we need the default value? Since there are default values for `background.position` and `background.size`

Reply:
it could probably be removed so that it's just specified in one place, I don't know which is best

Reply:
No we should not have it in two places. I suggest removing the default values here.

Reply:
Yea I agree. Keeping it aligned with the other styling defaults.